### PR TITLE
[Docs] Update docstrings

### PR DESCRIPTION
**Pull request by george0st**
Update a few doc strings

### CODE REVIEW

File: mlrun/db/httpdb.py [hunk 1]
```diff
@@ -147,6 +147,7 @@ def api_call(
         :param method: REST method (POST, GET, PUT...)
         :param path: Path to endpoint executed, for example ``"projects"``
         :param error: Error to return if API invocation fails
+        :param params:  TBD.
```
Comment (Tankilevitch) [inline mlrun/db/httpdb.py:150]:
```suggestion
        :param params:  Rest parameters, passed as a dictionary: ``{"<param-name>": <"param-value">}``
```

File: mlrun/model_monitoring/helpers.py [hunk 2]
```diff
@@ -117,6 +117,7 @@ def get_model_monitoring_batch_function(
     :param project:                     project name.
     :param model_monitoring_access_key: access key to apply the model monitoring process.
     :param db_session:                  A session that manages the current dialog with the database.
+    :param auth_info:                   TBD.
```
Comment (Tankilevitch) [inline mlrun/model_monitoring/helpers.py:120]:
```suggestion
    :param auth_info:                   The auth info of the request.
```

File: mlrun/run.py [hunk 3]
```diff
@@ -154,6 +154,8 @@ def run_local(
     :param params:   input parameters (dict)
     :param inputs:   input objects (dict of key: path)
     :param artifact_path: default artifact output path
+    :param mode:    TBD.
+    :param allow_empty_resources:   TBD.
```
Comment (Tankilevitch) [inline mlrun/run.py:158]:
```suggestion
    :param mode:    Runtime mode for more details head to `mlrun.new_function`.
    :param allow_empty_resources:   Allow passing non materialized set/vector as input to jobs (Allows to have function which don't depends on having targets, e.g a function which accepts a feature vector uri and generate the offline vector (parquet_ for it if it doesn't exists.
```

File: tests/system/demos/horovod/assets/utils_functions.py [hunk 4]
```diff
@@ -30,7 +30,7 @@ def open_archive(context, archive_url: DataItem, target_path, refresh=False):
     :param context:      function execution context
     :param archive_url:  url of archive file
     :param target_path:  file system path to store extracted files
-    :param key:          key of archive contents in artifact store
+    :param refresh: TBD.
```
Comment (Tankilevitch) [inline tests/system/demos/horovod/assets/utils_functions.py:33]: I think the `refresh` param can be removed it is not being used in the test

File: mlrun/runtimes/sparkjob/abstract.py [hunk 5]
```diff
@@ -280,6 +280,7 @@ def deploy(
         :param watch:      wait for the deploy to complete (and print build logs)
         :param with_mlrun: add the current mlrun package to the container build
         :param skip_deployed: skip the build if we already have an image for the function
+        :param is_kfp:  TBD.
```
Comment (Tankilevitch) [inline mlrun/runtimes/sparkjob/abstract.py:283]:
```suggestion
        :param is_kfp:  whether running inside KFP step ( No need to use, being enriched automatically if running inside one .
```

**Review by Tankilevitch — CHANGES_REQUESTED** [review]
@george0st Thanks for finding those missing docs, added comments for those params which you defined as TBD

**Comment by george0st** [comment]
@liranbg , please merge.

**Review by Tankilevitch — APPROVED** [review]

**Pull request merged**